### ISSUE TITLE
Assertion `fd > -1' failed

### ISSUE THREAD

**Issue by Unitech** [joyent/libuv#1207]
Since node 0.11.12 this errors happens to any pm2 command : 

```
../deps/uv/src/unix/core.c:431: uv__close: Assertion `fd > -1' failed.
```

https://github.com/Unitech/pm2/issues/354

Do you have any clue about this failed assertion ?

Thanks

**Comment by indutny** [comment]
Do you know any way to reproduce it?

**Comment by indutny** [comment]
Also, what OS?

**Comment by Unitech** [comment]
I'm under Ubuntu 12.04, using node 0.11.12.

To reproduce it, just install pm2 `npm install pm2@latest -g` and do a simple `pm2 list` and the error will be thrown

**Comment by saghul** [comment]
Can you please try applying https://github.com/joyent/libuv/pull/1210 ? I tried myself but I'm not sure I tested it correctly, since I couldn't manage to reproduce the issue.

**Comment by saghul** [comment]
Ping

**Comment by saghul** [comment]
Closing, I'm going to assume the issue was fixed with #1210 :-)
